staging/vhost-device-gpu/src/virtio_gpu.rs

```diff
@@ -387,24 +383,28 @@ impl RutabagaVirtioGpu {
         })
     }
 
-    fn configure_rutabaga_builder(gpu_mode: GpuMode) -> RutabagaBuilder {
-        let component = match gpu_mode {
+    fn configure_rutabaga_builder(gpu_config: &GpuConfig) -> RutabagaBuilder {
+        let component = match gpu_config.gpu_mode {
             GpuMode::VirglRenderer => RutabagaComponentType::VirglRenderer,
             #[cfg(feature = "gfxstream")]
             GpuMode::Gfxstream => RutabagaComponentType::Gfxstream,
         };
-        RutabagaBuilder::new(component, 0)
-            .set_use_egl(true)
-            .set_use_gles(true)
-            .set_use_glx(true)
-            .set_use_surfaceless(true)
+
+        RutabagaBuilder::new(component, gpu_config.capsets().bits())
+            .set_use_egl(gpu_config.flags().use_egl)
+            .set_use_glx(gpu_config.flags().use_glx)
+            .set_use_gles(gpu_config.flags().use_gles)
+            .set_use_surfaceless(gpu_config.flags().use_surfaceless)
+            .set_use_render_server(gpu_config.requires_render_server())
```

Comment:
I think we need to add an option for this `set_use_vulkan(true)` and also make it default for the gfxstream mode to set it automatically if vulkan usage hasn't been set. also I think we should set this `set_use_vulkan(true)` for the virglrenderer venus capset.

Reply:
Right, I didn't see that before. since it's already enabled by rutabaga we can leave it as it is.